layers/+lang/purescript/packages.el

```diff
@@ -52,6 +54,7 @@
     (progn
       (spacemacs/register-repl 'psci 'psci "purescript")
       (add-hook 'purescript-mode-hook 'inferior-psci-mode)
+      (spacemacs/declare-prefix "ms" "psci")
```

Comment:
"ms" is the "repl" prefix by convention across other layers, even though the actual backends are diffferent